src/components/FolderList/FolderList.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react";
+import "./FolderList.css";
+import FolderAdd from "../../assets/icons/FolderAdd";
+import ShareIcon from "../../assets/icons/Share";
+import PenIcon from "../../assets/icons/Pen";
+import DeleteIcon from "../../assets/icons/Delete";
+import { BASE_API_URL } from "../../utils/apiConfig";
+
+function ActionItem({ icon: IconComponent, label }) {
+  return (
+    <div className="action-item">
+      <IconComponent />
+      <span>{label}</span>
+    </div>
+  );
+}
+
+function FolderList({ onFolderSelect }) {
+  const [folders, setFolders] = useState([]);
+  const [selectedFolder, setSelectedFolder] = useState("전체");
+  const userId = 1;
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await fetch(`${BASE_API_URL}/users/${userId}/folders`);
+        if (response.status === 200) {
+          const data = await response.json();
```

Comment:
```suggestion
          const { data } = await response.json();
```
여기서 한번 destruction해주면 밑에서 data.data로 접근해줄 필요가 없어지겠죠?  또한 data가 못 오는 경우에 state update되는것을 대비해 data가 있을때만 state update되게끔 바꿔주시면 좋을 것 같아요!